test/connection.test.ts

```diff
@@ -193,7 +193,143 @@ describe('Connection', function () {
   })
 
   describe('Exchange Rate Handling', function () {
+    it('should modify the exchange rate until its worse than the minimum acceptable amount and the packets are rejected', async function () {
```

Comment:
This description should describe the behavior being tested rather than the behavior of the test itself

I would suggest something like "should reject and retry packets if the exchange rate is worse than the minimum acceptable rate"

Reply:
Updated to describe whats being tested.